generators/server/templates/src/test/java/package/web/rest/_LogsResourceIntTest.java

```diff
@@ -52,7 +52,7 @@
  */
 @RunWith(SpringRunner.class)
 <%_ if (authenticationType === 'uaa' && applicationType !== 'uaa') { _%>
-@SpringBootTest(classes = {<%= mainClass %>.class, SecurityBeanOverrideConfiguration.class})
+@SpringBootTest(classes = {SecurityBeanOverrideConfiguration.class, <%= mainClass %>.class})
```

Comment:
the new Spring Boot 2 Test seems to be order sensitive when loading the configuration files